gitclone/checkout_helper.go

```diff
@@ -123,6 +135,23 @@ func mergeWithCustomRetry(gitCmd git.Git, arg string, retry fallbackRetry) error
 	return nil
 }
 
+func fetchAndMerge(gitCmd git.Git, fetchParam fetchParams, mergeParam mergeParams) error {
+	headBranchRef := branchRefPrefix + fetchParam.branch
+	if err := fetch(gitCmd, fetchParam.remote, &headBranchRef, fetchParam.options); err != nil {
+		return nil
+	}
+
+	return mergeWithCustomRetry(gitCmd, mergeParam.arg, mergeParam.fallback)
+}
+
+func addForkRemote(gitCmd git.Git, repoURL string) error {
```

Comment:
does it worth wrapping `gitCmd.RemoteAdd` with this function?

Reply:
I agree, not worth being in a separated function.